src/Matrix.jl

```diff
@@ -1663,10 +1663,7 @@ function content(x::MatrixElem{T}) where T <: RingElement
   d = zero(base_ring(x))
   for i = 1:nrows(x)
      for j = 1:ncols(x)
-        d = gcd(d, x[i, j])
-        if isone(d)
```

Comment:
Did you remove `isone(d) && return d` on purpose? Perhaps it should be added to all these methods? I imagine it'll trigger quite frequently...

Reply:
the problem is the wierd gcd for Q: it is designed so that dividing by it leaves you with a primitive integral vector/ object. gcd(1, 1, 1/2) is 1/2, so dividing by it gives (2, 2, 1) which is integral and primitive.
I do no like it, but it has a useful use case and seems to match the julia convention for rational.
I prefer gcds of field elements to be 1. If I need a special thing (make integral and primitive), I'd rather have a different name. 
But I lost this argument before...